db/collection.go

```diff
@@ -526,6 +529,283 @@ func validateUpdateSchemaFields(
 	return hasChanged, nil
 }
 
+func (db *db) patchCollection(
```

Comment:
suggestion: I would much rather than patch stuff be moved to a new file, like: `db/collection_patch.go`

Reply:
I've opened a ticket: https://github.com/sourcenetwork/defradb/issues/2407